ui/lib/core/addon/components/toolbar-link.hbs

```diff
@@ -1,4 +1,7 @@
-{{#let (component "link-to" class="toolbar-link" disabled=@disabled params=@params) as |LinkToComponent|}}
+{{#let
+  (component "link-to" class="toolbar-link" disabled=@disabled route=this.route models=this.models)
```

Comment:
Not sure if you can answer this, but  I'm curious why we use the `component` helper here instead of using `<LinkTo>` below?

Reply:
Good question and I was wondering the same! In 4.4 I noticed that the class 'toolbar-link' was not being added to `LinkTo` when using the component helper so something has changed and this will need to be updated. I have a task to handle some LinkTo related things and am planning to update it there.